scripts/install.sh

```diff
@@ -187,6 +192,11 @@ oAuth2ApiEndpoint="undefined"
 patientPickerEndpoint="undefined"
 stage="dev"
 region="us-west-2"
+lambdaLatencyThreshold=3000
+apigatewayServerErrorThreshold=3
+apigatewayBadRequestErrorThreshold=5
+lambdaErrorThreshold=1
+DDBToESLambdaErrorThreshold=1
```

Comment:
Let's be consistent with our naming. I think you want this:
```suggestion
ddbToESLambdaErrorThreshold=1
```